src/DependencyInjection/HttplugExtension.php

```diff
@@ -292,6 +294,28 @@ private function configurePluginByName($name, Definition $definition, array $con
 
                 break;
 
+            case 'throttle':
+                if (!\interface_exists(LimiterInterface::class)) {
```

Comment:
this check is redundant. the throttle-plugin repository depends on symfony/rate-limiter. checking for the ThrottlePlugin is enough.

Reply:
Fixed